recontext/context.go

```diff
@@ -0,0 +1,29 @@
+// Provides means of obtaining a derived context which ignores parent's deadline, timeout, and cancellation.
```

Comment:
Docstrings have to be formatted "package context ......"

Reply:
done